### PR TITLE
CMD - service list command refactoring Fix #744

### LINKED ISSUE #744: Print service ID in cilium service list
```
cilium service list
10.0.0.10:53 =>
		1 => 10.1.0.3:53 (1)
10.0.0.131:80 =>
		1 => 10.15.148.179:80 (4)
		2 => 10.15.226.3:80 (4)
10.0.0.1:443 =>
		1 => 10.0.2.15:8443 (2)
```

- [ ] print proper header to explain flag, ID, backend IPs
- [ ] use `text/tabwriter`
- [ ] consider a more compact format to print backend IPs

### PR DESCRIPTION
**Pull request by eloycoto**
Hello! 

A few changes here: 

- [x] Added text/tabwriter
- [x] Changed the sort with the new options added in Golang 1.8. 
- [x] Added header labels

Around the compact format for backend address, I tried a different ways, but I didn't get any better option, if you have any suggestion I'm happy to change it. 

Current output: 
```
➜  cilium git:(744) ✗ ./cilium service list
ID   Frontend       Backend
2    0.0.0.0:90     1 => 192.168.100.1:80
                    2 => 192.168.100.2:80
                    3 => 192.168.100.4:80
3    0.0.0.0:8080   1 => 192.168.99.1:80
                    2 => 192.168.99.2:80
```

Regards.

### CODE REVIEW

File: cilium/cmd/service_list.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/cilium/cilium/common/types"
 
 	"github.com/spf13/cobra"
+	"text/tabwriter"
 )
```
Comment (aalemayhu) [inline cilium/cmd/service_list.go]: This import needs to be further up. Cilium and go imports should be split.

**Review by aalemayhu — CHANGES_REQUESTED** [review]
Other than the import, looks good.

**Comment by aalemayhu** [comment]
So the build is failing because the go version in the CI is `go1.7.4`.
@aanm shouldn't the VM be running at least `go1.8.1` by now?

```
[Runtime Tests] ==> cilium-master: cilium/cmd/service_list.go:90: undefined: sort.Slice
[Runtime Tests] ==> cilium-master: Makefile:43: recipe for target 'tests-consul' failed
[Runtime Tests] ==> cilium-master: make: Leaving directory '/home/vagrant/go/src/github.com/cilium/cilium'
[Runtime Tests] ==> cilium-master: make: *** [tests-consul] Error 1
```

File: cilium/cmd/service_list.go (hunk 1 above)
Comment (eloycoto) [reply cilium/cmd/service_list.go]: Sorry! It is now changed.

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
> @aanm shouldn't the VM be running at least go1.8.1 by now?

@tgraf was the new image pushed to atlas?

**Comment by tgraf** [comment]
> @tgraf was the new image pushed to atlas?

It's currently building on my mac. I've updated the scripts a bit as well: https://github.com/cilium/packer-ubuntu-16.10/pull/4

**Review by aalemayhu — APPROVED** [review]

**Comment by tgraf** [comment]
> It's currently building on my mac. I've updated the scripts a bit as well: cilium/packer-ubuntu-16.10#4

New vagrant box is uploaded and updated on Jenkins slaves. Please `rebase master --ignore-date` and `push -f` to trigger a new Jenkins run.

**Comment by eloycoto** [comment]
Hey, 

I updated the branch from master. Looks like that there is a problem with the labels. Not sure if something related to this branch. 

Do I need to do anything here?

Best regards

**Comment by tgraf** [comment]
> Do I need to do anything here?

No, this seems to be an issue with upgrading to 1.8.3, we're tracking this in #955

**Comment by eloycoto** [comment]
Hello!

Ok, I'll keep an eye on that issue. I forgot to check the tests suite and 06-lb.sh is affected. I changed a few lines, and I use the REST API instead of use the cilium list, not sure if It is correct, if you're not happy with that I'll change to the original behaviour ;-)

**Comment by tgraf** [comment]
> Ok, I'll keep an eye on that issue. I forgot to check the tests suite and 06-lb.sh is affected. I changed a few lines, and I use the REST API instead of use the cilium list, not sure if It is correct, if you're not happy with that I'll change to the original behaviour ;-)

The CI is back, you can push again after you've fixed the issues of this PR.

**Comment by eloycoto** [comment]
All looks good now, I think that is good to merge ;-)

**Pull request merged**